operator/bundle/community-openshift/manifests/loki-operator.clusterserviceversion.yaml

```diff
@@ -1771,7 +1771,7 @@ spec:
           - update
           - watch
         - apiGroups:
-          - policy
+          - policy/v1
```

Comment:
I believe policy only became stable with `1.21` do we should bump our compatibility to `1.21` from `1.16`

Reply:
You mean `minKubeVersion`?

Reply:
I didn't realise `minKubeVersion` I actually meant `docs/operator/compatibility.md` that still references 1.16